src/inspector_socket.cc

```diff
@@ -92,11 +71,25 @@ static void close_connection(inspector_socket_t* inspector) {
   }
 }
 
+struct WriteRequest {
+  WriteRequest(inspector_socket_t* inspector, const char* data, size_t size)
+      : inspector(inspector)
+      , storage(data, data + size)
+      , buf({ &storage[0], storage.size() }) {}
```

Comment:
This doesn’t work on Windows, the order of the members in `uv_buf_t` is reversed there.